Training/Program.cs

```diff
@@ -1,7 +1,14 @@
-namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+// Isogram checker  
+Console.Write ("Enter a string: ");                                       // Gets an input string 
+string isostr = Console.ReadLine ();
+static bool IsIsogram (string isostr) {                                   // A method which checks whether the input is isogram or not
+   char[] result = isostr.ToCharArray ();                                 // Spliting the string into single elements 
+   Array.Sort (result);                                                    // Sorting the elements in ascending order 

```

Comment:
Can you please tell how sorting is done here?

Reply:
The input string is taken and converted into a character array. After that, we sort the characters in ascending order.